.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+  release:
+    types: published
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install dependencies
+      run: |
+        sudo apt install -y python3-pytest
+        curl -L -O https://github.com/Kitware/CMake/releases/download/v3.16.2/cmake-3.16.2-Linux-x86_64.sh
```

Comment:
might consider `pip install cmake` instead

Reply:
Can I install a specific CMake version with pip?

Reply:
Can I install a specific CMake version with pip?

Reply:
Yes you can pick from the CMake versions at: https://pypi.org/project/cmake/#history

`pip install cmake==3.16.2` and so on

Reply:
Neat! Thanks :)